docs/index.yaml

```diff
@@ -0,0 +1,378 @@
+AWS:
+    Credential Access:
```

Comment:
Let's quote this to make sure we have no YAML parsing issues

Reply:
This can be parsed just fine. The specification allows fields (and values) with spaces: https://yaml.org/

If you try to parse the `index.yaml` into the `map[stratus.Platform]map[string][]*stratus.AttackTechnique`, it fails because of the custom `MarshalYAML` function added to the `Tactic` struct - that functions gets the `int` value and transforms into a `string`. That can be solved by adding the `UnmarshalYAML` function doing the reverse operation. If you would like I can add this function as well.

```go
func (t Tactic) UnmarshalYAML(node *yaml.Node) error {
	value := node.Value
	t, err := AttackTacticFromString(value)
	return err
}
```